test/Microsoft.ML.Sweeper.Tests/SweeperTest.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Sweeper;
+using System;
+using System.IO;
+using Xunit;
+
+namespace Microsoft.ML.Sweeper.Tests
+{
+    public class SweeperTest
+    {
+        [Fact]
+        public void UniformRandomSweeperReturnsDistinctValuesWhenProposeSweep()
+        {
+            DiscreteValueGenerator valueGenerator = CreateDiscreteValueGenerator();
+
+            using (var writer = new StreamWriter(new MemoryStream()))
+            using (var env = new TlcEnvironment(42, outWriter: writer, errWriter: writer))
+            {
+                var sweeper = new UniformRandomSweeper(env,
+                    new SweeperBase.ArgumentsBase(),
+                    new[] { valueGenerator });
+
+                var results = sweeper.ProposeSweeps(5000);
+                Assert.NotNull(results);
+
+                int length = results.Length;
+                Assert.Equal(2, length);
+            }
+        }
+
+        [Fact]
+        public void RandomGridSweeperReturnsDistinctValuesWhenProposeSweep()
+        {
+            DiscreteValueGenerator valueGenerator = CreateDiscreteValueGenerator();
+
+            using (var writer = new StreamWriter(new MemoryStream()))
+            using (var env = new TlcEnvironment(42, outWriter: writer, errWriter: writer))
+            {
+                var sweeper = new RandomGridSweeper(env,
+                    new RandomGridSweeper.Arguments(),
+                    new[] { valueGenerator });
+
+                var results = sweeper.ProposeSweeps(5000);
```

Comment:
>5000 [](start = 52, length = 4)

is 5000 just an arbitrary big number?Can we get by with something small, like 3,  since we are setting the seed for random number generations on the TlcEnvironment.

Reply:
Yes 5000, was just a bit number - I have amended to 3 and the tests still pass